packages/adapter-kysely/src/database.ts

```diff
@@ -0,0 +1,46 @@
+import type { Generated } from "kysely"
+
+interface User {
+  id: Generated<string>
+  name: string | null
+  email: string
+  emailVerified: Date | string | null
+  image: string | null
+}
+
+interface Account {
+  id: Generated<string>
+  userId: string
+  type: string
+  provider: string
+  providerAccountId: string
+  refresh_token: string | null
+  access_token: string | null
+  expires_at: number | null
+  token_type: string | null
+  scope: string | null
+  id_token: string | null
+  session_state: string | null
+  oauth_token_secret: string | null
+  oauth_token: string | null
+}
+
+interface Session {
+  id: Generated<string>
```

Comment:
If id's are meant to be read-only, should use `GeneratedAlways` instead of `Generated`.

```suggestion
import type { GeneratedAlways } from "kysely"

interface User {
  id: GeneratedAlways<string>
  name: string | null
  email: string
  emailVerified: Date | string | null
  image: string | null
}

interface Account {
  id: GeneratedAlways<string>
  userId: string
  type: string
  provider: string
  providerAccountId: string
  refresh_token: string | null
  access_token: string | null
  expires_at: number | null
  token_type: string | null
  scope: string | null
  id_token: string | null
  session_state: string | null
  oauth_token_secret: string | null
  oauth_token: string | null
}

interface Session {
  id: GeneratedAlways<string>
```